lib/ansiblelint/__init__.py

```diff
@@ -19,5 +19,11 @@
 # THE SOFTWARE.
 """Main ansible-lint package."""
 
-# deprecated backwards compability:
-from ansiblelint.rules import AnsibleLintRule  # noqa: F401 api backwards compability
+from ansiblelint.rules import AnsibleLintRule
+from ansiblelint.version import __version__
```

Comment:
honestly, I'm not sure if we want to expose it like this. I understand that it's common but earlier you advocated for removing everything from the package init. We need to have some sort of a decision of what we actually want to do with this.